tests/test_evaluation.py

```diff
@@ -0,0 +1,128 @@
+import mlflow
+
+from mlflow.models.evaluation import evaluate, EvaluationDataset
+import sklearn
+import sklearn.datasets
+import sklearn.linear_model
+import pytest
+import numpy as np
+
+from sklearn.metrics import (
+    accuracy_score,
+    confusion_matrix,
+    mean_absolute_error,
+    mean_squared_error,
+)
+
+from mlflow_test_plugin.dummy_evaluator import Array2DEvaluationArtifact
+from mlflow.tracking.artifact_utils import get_artifact_uri
+
+
+def get_iris():
+    iris = sklearn.datasets.load_iris()
+    return iris.data[:, :2], iris.target
+
+
+def get_run_data(run_id):
+    client = mlflow.tracking.MlflowClient()
+    data = client.get_run(run_id).data
+    # Ignore tags mlflow logs by default (e.g. "mlflow.user")
+    tags = {k: v for k, v in data.tags.items() if not k.startswith("mlflow.")}
+    artifacts = [f.path for f in client.list_artifacts(run_id)]
+    return data.params, data.metrics, tags, artifacts
+
+
+def get_local_artifact_path(run_id, artifact_path):
+    return get_artifact_uri(run_id, artifact_path).replace("file://", "")
+
+
+@pytest.fixture(scope="module")
+def regressor_model():
+    X, y = get_iris()
+    reg = sklearn.linear_model.LinearRegression()
+    reg.fit(X, y)
+    return reg
+
+
+@pytest.fixture(scope="module")
+def classifier_model():
```

Comment:
Might want to have another data set that can test binary classification as well as multiclass since the metrics are going to be different based on what is available in the labels. The standard binary classification data loaded in sklearn is the `sklearn.datasets.load_breast_cancer()` data